TabulateSmarterTestAdminPackage.Common/Validators/Validator.cs

```diff
@@ -15,6 +15,11 @@ protected Validator(ErrorSeverity errorSeverity, object parameter = null)
 
         public abstract bool IsValid(object value);
 
+        public bool IsValid(object value, bool isRequired)
+        {
+            return IsValid(value) || !isRequired && value == null;
```

Comment:
I'll drop the second conditional onto its own line. That's consistent with the formatting style i'm using elsewhere